File: CPP/Aksesoris.cpp
#include "Petshop.cpp"

using namespace std;

class Aksesoris : public PetShop
{
private:
  string jenis, bahan, warna;

public:
  Aksesoris()
  {
    this->jenis = "";
    this->bahan = "";
    this->warna = "";
  }

  Aksesoris(int id, string nama, int harga, int stok, string jenis, string bahan, string warna)
  {
    this->setId(id);
    this->setNama(nama);
    this->setHarga(harga);
    this->setStok(stok);
    this->jenis = jenis;
    this->bahan = bahan;
    this->warna = warna;
  }

  void setJenis(string jenis)
  {
    this->jenis = jenis;
  }

  void setBahan(string bahan)
  {
    this->bahan = bahan;
  }

  void setWarna(string warna)
  {
    this->warna = warna;
  }

  string getJenis()
  {
    return this->jenis;
  }

  string getBahan()
  {
    return this->bahan;
  }

  string getWarna()
  {
    return this->warna;
  }
};

File: PHP/Aksesoris.php
<?php
  require_once 'PetShop.php';

class Aksesoris extends PetShop
{
  private $jenis = "";
  private $bahan = "";
  private $warna = "";

  public function __construct($id, $nama, $harga, $stok, $jenis, $bahan, $warna)
  {
    parent::__construct($id, $nama, $harga, $stok);
    $this->jenis = $jenis;
    $this->bahan = $bahan;
    $this->warna = $warna;
  }

  public function getJenis()
  {
    return $this->jenis;
  }

  public function getBahan()
  {
    return $this->bahan;
  }

  public function getWarna()
  {
    return $this->warna;
  }

  public function setJenis($jenis)
  {
    $this->jenis = $jenis;
  }

  public function setBahan($bahan)
  {
    $this->bahan = $bahan;
  }

  public function setWarna($warna)
  {
    $this->warna = $warna;
  }

}


?>